com.ibm.wala.core/build.gradle.kts

```diff
@@ -72,11 +72,7 @@ dependencies {
   )
 }
 
-val extraJavadocClasspath: Configuration by configurations.creating { isCanBeConsumed = false }
-
-dependencies { extraJavadocClasspath(project(":com.ibm.wala.dalvik")) }
-
-tasks.named<Javadoc>("javadoc") { classpath += extraJavadocClasspath }
+dependencies { javadocClasspath(project(":com.ibm.wala.dalvik")) }
```

Comment:
Can we at least file a follow-up task to figure out why this is necessary?  It seems really wrong to me that building Javadoc for core depends on Dalvik somehow

Reply:
Yes, certainly. Or if `dalvik` is required here, then we can at least add some documentation, right here in this build logic, to explain why. I'll look into that in a separate PR to follow the current one.

Reply:
#1189 documents my intention to follow up on @msridhar's concerns soon.